module/VuFind/src/VuFind/Search/Factory/EPFBackendFactory.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Factory for EDS backends used for EPF.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2013.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   David Maus <maus@hab.de>
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Search\Factory;
+
+/**
+ * Factory for EDS backends used for EPF.
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   David Maus <maus@hab.de>
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class EPFBackendFactory extends EdsBackendFactory
+{
+    /**
+     * Get the service name.  This is used for both configuration
+     * and record driver retrieval.
+     *
+     * @return str
+     */
+    protected function getServiceName()
+    {
+        return 'EPF';
+    }
+
+    /**
+     * Create the options array for the EDS connector.
+     *
+     * @return array
+     */
+    protected function createConnectorOptions()
+    {
+        $options = parent::createConnectorOptions();
+        $options['api_url'] = $this->edsConfig->General->api_url
+            ?? 'https://eds-api.ebscohost.com/edsapi/publication';
```

Comment:
Maybe you could use a $defaultApiUrl property in the factory to allow this customization, instead of having to override api_url here.